rcsb/workflow/targets/ProteinTargetSequenceExecutionWorkflow.py

```diff
@@ -114,7 +128,8 @@ def exportFasta(self):
         ok = False
         try:
             ptsW = ProteinTargetSequenceWorkflow(self.__cfgOb, self.__cachePath)
-            ok = ptsW.exportTargetsFasta(useCache=True, addTaxonomy=True, reloadPharos=True, fromDbPharos=True, resourceNameList=["sabdab", "card", "drugbank", "chembl", "pharos"])
+            ok = ptsW.exportTargetsFasta(useCache=True, addTaxonomy=True, reloadPharos=True, fromDbPharos=True,
+                                         resourceNameList=["sabdab", "card", "drugbank", "chembl", "pharos"])
```

Comment:
I'm surprised pylint let you do that. Usually the formatting requires that for multiline parentheses, it should look like:
```python
ok = ptsW.exportTargetsFasta(
    useCache=True, addTaxonomy=True, reloadPharos=True, fromDbPharos=True, resourceNameList=["sabdab", "card", "drugbank", "chembl", "pharos"]
)
```

For consistency purposes with other projects, do you mind reformatting that?

Reply:
(as well as the other methods here?)

Another thing I thought it usually requires is that when you give multiple arguments that extend beyond the 185 character limit, you must split them into a line per argument—not, say, 5 arguments on line one and 5 arguments on a second line. I know it's yelled at me for that before, so I'm surprised it's passing for you now.

Reply:
Hmm, maybe I don't have pylint configured correctly on my IDE, I'll take a look.

Reply:
Fixed the sloppy wrapping and reverted the unnecessary wrapping. Turns out my IDE guide was also set to 120, not 185.